src/App.js

```diff
@@ -51,14 +51,12 @@ function App() {
             <Redirect to={ROUTES.CONTACTS} />
           </Route>
           <Route path={ROUTES.CONTACTS}>
-             {/* Add props to ContactsPage */}
-            <ContactsPage contacts={contacts} addContact={addContact}/>  
+            <ContactsPage contacts={contacts} addContacts={addContacts}/>  

```

Comment:
Please keep this as addContact singular, it will affect the rest of the code.

Reply:
- changed addContacts to addContact